<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VetementForm</class>
 <widget class="QDialog" name="VetementForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VetementForm</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Prix</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>58</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Code</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Taille</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Couleur</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditDesc">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Chandail</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPrix">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>57.99</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCode">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>110</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>PRD-CHAN-44</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditCouleur">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>vert</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxTaille">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>140</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>XS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>S</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>M</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>L</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>XL</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>XXL</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonOK</sender>
   <signal>clicked()</signal>
   <receiver>VetementForm</receiver>
   <slot>slotOK()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>189</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotOK()</slot>
 </slots>
</ui>
